.gitignore

```diff
@@ -23,3 +23,4 @@ _[Rr]e[Ss]harper.*/
 src/_NCrunch_Nancy/
 Thumbs.db
 .idea
+/src/.vs
```

Comment:
I'm not the maintainer of this repo, but I am of other projects, in general I would try to change as little as possible and keep commits cleans, in general I wouldn't touch repo/project settings when adding a feature(unless that's required in someway)